officemix/officemix.py

```diff
@@ -105,6 +109,16 @@ def studio_submit(self, data, suffic=''):
         
         return {'result': 'success'}
 
+    @XBlock.json_handler
+    def emit_event(self, data, suffix=''):
```

Comment:
why not use https://github.com/edx/xblock-utils/blob/master/xblockutils/publish_event.py?